lib/common_test/src/ct_hooks.erl

```diff
@@ -36,7 +36,7 @@
 %% All of the hooks which are to be started by default. Remove by issuing
 %% -enable_builtin_hooks false to when starting common test.
 -define(BUILTIN_HOOKS,[#ct_hook_config{ module = cth_log_redirect,
-					opts = [],
+					opts = [{mode, add}],
```

Comment:
needed if it's default?

Reply:
I moved it out of here and into the `proplists:get_value` call in 1b8bab7a77e7e7a3319d6bd48f4ab9fa4ea88a1a.